src/Components/Page/PageContents.tsx

```diff
@@ -17,6 +20,10 @@ export interface PageContentsProps extends StandardFunctionProps {
   autoHideScrollbar?: boolean
   /** Controls the gutters (left and right margins) */
   gutters?: ComponentSize
+  /** Exposes custom onScroll handler */
+  onScroll?: FusionScrollHandler
+  /** Allows user to customize visible height of container */
+  visibleHeight?: string
```

Comment:
Couldn't we just define `min-height: 100px` in the CSS?

Reply:
We could, but i want the user to be able to pass one.